CMakeLists.txt

```diff
@@ -398,7 +419,29 @@ else()
 	)
 endif()
 
-install(FILES ${AdditionalInstallFiles} DESTINATION ${auxresourcesdir})
+if(WIN32)
+		SET(CMAKE_INSTALL_SYSTEM_RUNTIME_DESTINATION "${CMAKE_INSTALL_PREFIX}/${auxresourcesdir}")
+		include(InstallRequiredSystemLibraries)
+
+		install(FILES ${AdditionalInstallFiles} DESTINATION "${CMAKE_INSTALL_PREFIX}/${auxresourcesdir}")
+
+	if(MSYS)
+		SET(QT_PLATFORM_PLUGIN_PREFIX ${Qt5Core_DIR}/../../../share/qt5/plugins/platforms CACHE STRING "Location of qt platform plugin for windows. Override when not using MinGW-cmake!")
+	else()
+		SET(QT_PLATFORM_PLUGIN_PREFIX ${Qt5Core_DIR}/../../../plugins/platforms CACHE STRING "Location of qt platform plugin for windows. Override if you have multiple paths in your cmake prefix path")
+	endif()
+
+	install(FILES "${QT_PLATFORM_PLUGIN_PREFIX}/qwindows.dll"
+		CONFIGURATIONS Release
+		DESTINATION "${CMAKE_INSTALL_PREFIX}/${auxresourcesdir}/platforms"
+	)
+	install(FILES "${QT_PLATFORM_PLUGIN_PREFIX}/qwindowsd.dll"
+		CONFIGURATIONS Debug
+		DESTINATION "${CMAKE_INSTALL_PREFIX}/${auxresourcesdir}/platforms"
+	)
```

Comment:
this should go into `editors/scide/CMakeLists.txt`, to the same place where the platform plugins for cocoa are deployed.
adding a post-build step would also make the non-installed binaries runnable